src/k8s/cmd/k8sd/k8sd_cluster_recover.go

```diff
@@ -104,22 +104,24 @@ func newClusterRecoverCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "cluster-recover",
 		Short: "Recover the cluster from this member if quorum is lost",
-		RunE: func(cmd *cobra.Command, args []string) error {
+		Run: func(cmd *cobra.Command, args []string) {
 			log.Configure(log.Options{
 				LogLevel:     rootCmdOpts.logLevel,
 				AddDirHeader: true,
 			})
 
 			if err := recoveryCmdPrechecks(cmd.Context()); err != nil {
-				return err
+				cmd.PrintErrf("Recovery precheck failed: %v\n", err)
+				os.Exit(1)
```

Comment:
maybe it's better to use `env.Exit` here and in the other places as well.

p.s. I noticed that `env` is not passed to the `newClusterRecoverCmd`. maybe we can do this part (passing `env` and using `env.Exit`) in a follow up PR.

Reply:
Right away, to be honest I haven't noticed the ExecutionContext abstraction.

I'll update the commit.